server/events/project_command_context_builder.go

```diff
@@ -2,14 +2,19 @@ package events
 
 import (
 	"path/filepath"
-	"regexp"
+	"sort"
 
-	"github.com/hashicorp/go-version"
+	"github.com/Masterminds/semver"
 	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+
+	"github.com/hashicorp/go-version"
 	"github.com/runatlantis/atlantis/server/events/models"
 	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+	lib "github.com/warrensbox/terraform-switcher/lib"
 )
 
+var mirrorURL = "https://releases.hashicorp.com/terraform"
```

Comment:
I think this could be a configurable URL because some companies host TF exe in their private repos for setups with no public access and/or to control which TF versions are currently cleared to be used.

Reply:
Created issue https://github.com/runatlantis/atlantis/issues/2701